standalone/include/psen_scan_v2_standalone/data_conversion_layer/laserscan_conversions.h

```diff
@@ -160,7 +160,7 @@ LaserScanConverter::thetaAnglesFitTogether(const std::vector<data_conversion_lay
   for (auto index : sorted_frames_indices)
   {
     const auto& frame = frames[index];
-    if (lastEnd != frame.fromTheta())
+    if ((lastEnd != frame.fromTheta()) && (!frame.measurements().empty()))
```

Comment:
Did you try it out? If the previous frame was empty, there should also be a mismatch.

Reply:
You are right, Thnx